vllm/config/model.py

```
@@ -1183,6 +1183,15 @@ def _get_runner_type(
         convert: ConvertOption,
     ) -> RunnerType:
         if runner != "auto":
+            if runner == "pooling":
+                if not self.registry.is_pooling_model(architectures, self):
```

Ruff (0.16.3)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)